src/com/github/adambots/steamworks2017/intake/Intake.java

```diff
@@ -0,0 +1,35 @@
+package com.github.adambots.steamworks2017.intake;
+
+import org.usfirst.frc.team245.robot.Actuators;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Intake {
+	
+	//TODO: Set MAX_CURRENT to a number once we get an exact value
+	public static double MAX_CURRENT;
+	public static double SPEED = 0.8;
+	
+	/*
+	 * Runs intakeMotor
+	 * @Param intakeButon
+	 * */
+	public static void intake(boolean intakeButton){
+		if (intakeButton == true){
```

Comment:
If you're just checking a boolean you can write if(myBool) rather than if(myBool == true). Same for writing if(!myBool) instead of if(myBool == false)